src/foodListing/index.html

```diff
@@ -3,17 +3,20 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Healthy Bites| List of the foods depending of the type</title>
+    <title>Healthy Bites</title>
     <link rel="stylesheet" href="../css/style.css">
+       <script type="module" src="../js/foodListing.js"></script>
 </head>
 <body>
-    <header class="divider" id="main-heaer"></header>
+    <header class="divider" id="main-header"></header>
 
     <main>
-            <!-- RENDER THE RECEPIES BY CATEGORY {DINNER - BREAKFAST - DRINKS - MAIN COURSE}-->
-    </main>
+        <ul class="recipe-listing">
 
+        </ul>
+        This is Food listing Page</main>
     <footer id="main-footer"></footer>
-    <script src="../js/foodListing.js" type="module"></script>
+
+ 
```

Comment:
@JosueBCe if you'll be loading this in through the router you'll want to remove everything but the `<main>` probably (I didn't change this file)